wandb/internal/internal.py

```diff
@@ -200,7 +200,17 @@ def _check_process(settings, pid):
         os._exit(-1)
 
 
-def wandb_internal(
+def _mock_requests():
+    """Mocks all server requests to `mock_server.py` used in tests"""
```

Comment:
This is controversial.  I'm pulling in our test utils here and mocking the backend server process.  A better solution may be to not start the process at all and create a mock backend that responds the same as our mock_server...

Reply:
you arent using this anymore, can we remove.  mocking the backend or interface is preferable.